tools/pythonpkg/tests/fast/api/test_to_csv.py

```diff
@@ -238,7 +238,8 @@ def test_to_csv_per_thread_output(self, pandas):
         rel = duckdb.from_df(df)
         rel.to_csv(temp_file_name, header=True, per_thread_output=True)
         csv_rel = duckdb.read_csv(f'{temp_file_name}/*.csv', header=True)
-        assert rel.execute().fetchall() == csv_rel.execute().fetchall()
+        # Skip test due to unreliablility on certain platforms
```

Comment:
Thanks, that's not how we skip tests though
Please add this decorator to the function definition:
```py
@pytest.mark.skip(reason="Skip test due to unreliablility on certain platforms")
```

Reply:
> CSV one has been unreliable depending on Pyhton versions

In that case we can also use `skipif` here to only skip the test on that python version

Reply:
I have no idea which condition make this fail, and unsure how to parametrise a function twice.
Similar on pytorch version, I am not sure if that's the actual problem.

Reply:
I don't agree with commenting this out at all
That will just make it silently fail, hiding a potential problem, and nothing even indicating that it might be a problem

At least with a `pytest.mark.skip` (you can just add it on top, it has no problem with the existing parametrize decorator) it's deliberate and we can circle back to it at some point

Reply:
Added a skip, thanks

Reply:
I pushed <https://github.com/duckdb/duckdb/pull/11819>
I think that will fix the problem, unless this issue is consistent, in that case feel free to skip it until we figure it out

But I suspect it has to do with the error prone looking way that paths are created there

Reply:
I don't think that PR solves the problem, or not in total.